BUILD

```diff
@@ -0,0 +1,25 @@
+load("@io_bazel_rules_go//go:def.bzl", "gazelle", "go_binary", "go_library", "go_prefix")
+
+gazelle(
+    name = "gazelle",
+    external = "vendored",
```

Comment:
add in 

```bazel
    build_tags = [
        "container_image_ostree_stub",
        "containers_image_openpgp",
    ],
```

Reply:
> Should test.sh run bazel run //:gazelle? That would force any contributors to have the bazel toolchain installed.

I'm fine with that. Ultimately our downoading/installing of stuff in the minikube makefile causes extra confusion. We could try to move it into a container - but that gets tricky too...